porsche-digital-lab-berlin-blockchain/catbox
# Feat/share data with user after creation

## What does it do?
* allows for more dynamic usage of deployed contracts by using the new `deployments.json` file
* adds another high level function `shareEncryptedData`

## How to test?
1. Redeploy contracts in newly updated `catbox-contracts` repo
1. run `yarn test` in this repo
1. see that all tests pass
